docs/UserGuide.md

```diff
@@ -55,7 +55,12 @@ Interface (CLI) while still having the benefits of a Graphic User Interface (GUI
   e.g `n/NAME [s/CLIENT_SOURCE]` can be used as `n/John Doe s/friend` or as `n/John Doe`.
 
 * Items with `…`​ after them can be used multiple times including zero times.<br>
-  e.g. `[s/CLIENT_SOURCE]…​` can be used as ` ` (i.e. 0 times), `s/friend`, `s/friend s/family` etc.
+  e.g. `[s/CLIENT_SOURCE]…​` can be used as ` ` (i.e. 0 times), `s/friend`, `s/foe s/family` etc.
+
+* Items without `…`​ after them can also be used multiple times. 
+  However, only the last occurrence would be used as the input to the command.<br>
```

Comment:
The first line seems contradictory to the line before if it was just read alone. might make the reader confused.
Instead, perhaps we can say that:
If items with `...` after them was inputted multiple times, only the last occurrence would be used as the input to the command.